sql/information_schema/columns_table.go

```diff
@@ -190,7 +191,10 @@ func columnsRowIter(ctx *sql.Context, cat sql.Catalog, columnNameToDefault map[s
 				}
 
 				fullColumnName := db.Name() + "." + t.Name() + "." + c.Name
-				colDefault = trimColumnDefaultOutput(columnNameToDefault[fullColumnName])
+				colDefault = getColumnDefaultValue(ctx, columnNameToDefault[fullColumnName])
+				if colDefault != nil {
+					colDefault = fmt.Sprint(colDefault)
+				}
```

Comment:
(minor) You might consider moving this `if colDefault != nil` block into your `getColumnDefaultValue` helper method. It seems very tightly coupled to that logic, so might make sense to move it in there and keep this current method a few lines shorter.